sdk/storage/Azure.Storage.Files.DataLake/src/Models/AccessControlChangeFailure.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// Represents an entry that failed to update Access Control List.
+    /// </summary>
+    public struct AccessControlChangeFailure
```

Comment:
Why a struct?  Same for the others.

Reply:
https://apiview.dev/Assemblies/Conversation/05e4cdd03e3f43e4830d800a950a4dce